### PR TITLE
Change the defaults

### PR DESCRIPTION
**Pull request by takism1**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 telegraf_hostname:
 telegraf_omit_hostname: "false"
-telegraf_install_url:
 
 telegraf_influxdb_urls:
   - http://localhost:8086
 telegraf_influxdb_database: telegraf
-telegraf_influxdb_precision: s
-telegraf_influxdb_retention_policy: autogen
+telegraf_influxdb_precision: us
```
Comment (mmdenny) [inline defaults/main.yml]: Is there a reason we need microsecond precision?  Also, why change the retention policy?

File: defaults/main.yml [hunk 2]
```diff
@@ -37,31 +37,25 @@ telegraf_influxdb_user_agent:
...
+        - "devtmpfs"
   - name: diskio
+  - name: kernel
+  - name: mem
+  - name: processes
+  - name: swap
+  - name: system
+  - name: udp_listener
     options:
-      skip_serial_number: "true"
```
Comment (mmdenny) [inline defaults/main.yml]: Are these all to reflect what's in telegraf now in prod?

File: defaults/main.yml (hunk 1 above)
Comment (takism1) [inline defaults/main.yml]: To reflect what's now in prod

File: tasks/install.yml [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+    - python-httplib2
+    - python-apt
+    - curl
+    - apt-transport-https
 
-- include: install-debian.yml
-  when: ansible_os_family == "Debian"
\ No newline at end of file
+- name: Download Telegraf package via URL [Debian/Ubuntu]
+  command: curl -o /tmp/telegraf-ansible-download.deb {{ telegraf_install_url }}
```
Comment (mmdenny) [inline tasks/install.yml]: @takism1 Is there a reason we're using curl in the to install the deb package instead of using the file install-debian.yml ?

**Comment by mmdenny** [comment]
@takism1 Other than my comment on curl above, lgtm.  How do you want to handle this?  Do you want to check this into master, then we'll merge from the root repo when we want to upgrade?

**Comment by MasterDDT** [comment]
Shouldn't this be merged?

**Comment by mrwiggles** [comment]
@takism1 Who's the owner here? I can't push to this, so...

**Comment by mrwiggles** [comment]
@takism1 Thanks to Nitay, we are all now set as owners, so we can actually get this PR in if necessary.

**Pull request merged**